docs/onboarding_to_cluster.md

```diff
@@ -97,14 +92,23 @@ users:
   - USER_2
 ```
 
-Now please encrypt the file with sops and verify it contains a fingerprint of both GPG keys: the OperateFirst master key and your team's key.
+> Note that the USER_n value is the email used to login via SSO in the preceeding steps.
+
+Encrypt the file with sops:
+
+```sh
+$ sops --encrypt --encrypted-regex="^users$" --pgp="0508677DD04952D06A943D5B4DC4116D360E3276" groups/GROUP_NAME.yaml > groups/GROUP_NAME.enc.yaml
```

Comment:
Can you please point to https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/moc#secret-management somewhere here? So they know where to import the keys from?

Reply:
done